jgeorge37/acmwosu-db
# Ada 53

To test:
1. Go to `http://dev-ada.herokuapp.com/execdashboard`
2. Click on "Create Account" button
3. If you type, people should show up. Try making an account from an existing student (may have to insert a student or delete an account for a student)
4. Try making a student and account for a new person.
5. Do 4. except the person is a substring of another person so search has matches. There should be a "New Student?" option that will make a new person regardless)
6. Mess around with the input to see errors messages.

Do `select * from student;` and `select * from account;` to check people are actually being made.
